scripts/translation-pipeline/CLAUDE.md

```
@@ -0,0 +1,74 @@
+
+# Build
+pnpm --filter @vapor-ui/translation-pipeline build
+```
+
+Path alias `~` maps to `src/`. (`~/types` → `src/types.ts`)
+
+## Architecture
+
+### Core Flow
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)